grunttasks/l10n-generate-pages.js

```diff
@@ -104,7 +104,7 @@ module.exports = function (grunt) {
     // for editors should be ignored.
     var templates = grunt.file.expand({
       cwd: templateSrc
-    }, '**/*.html');
+    }, ['**/*.html', '!style-guide.html']);
```

Comment:
Great! so this should get Travis passing, two things for this change:

Current templates that are copied are

```
templates [ '404.html',
  '500.html',
  '502.html',
  '503.html',
  'index.html',
  'mocha.html',
  'privacy.html',
  'terms.html' ]
```


1. Can we add `!mocha.html` into this array here as well? 
2. Add a comment above saying something like: "// Ignore the mocha and style-guide templates, they are only used for testing"